### ISSUE TITLE
Creating a new EditorSettings variable then run in a visual script reports SOCKET ERROR

### ISSUE THREAD

**Issue by haizhou37** [godotengine/godot#24605]
**Godot version:**
Godot3.1 Aplha4

**OS/device including version:**
manjaro gnome x64
Intel® Core™ i3-2350M CPU @ 2.30GHz × 4
Intel® Sandybridge Mobile
OpenGL ES 3.0 Renderer: Mesa DRI Intel(R) Sandybridge Mobile
intel hd 3000

**Issue description:**
![2018-12-26 21-41-14](https://user-images.githubusercontent.com/33272474/50447881-204a7280-0959-11e9-9025-7e8ba241f5fa.png)
Creating other variable is normal(I just tested a little bit. I can't rule out hiding anything else.).

**Steps to reproduce:**
Creating a new EditorSettings variable

**Minimal reproduction project:**
[test11.zip](https://github.com/godotengine/godot/files/2710345/test11.zip)

**Comment by KoBeWi** [comment]
The project crashes with these errors when run (3.2.2):
![image](https://user-images.githubusercontent.com/2223172/86534926-e9a1d080-bedc-11ea-980f-8bdb3dc8674b.png)
There's no backtrace for the crash.

Although seeing the EditorSettings error, I think this is somewhat expected and comes from user error?

**Comment by akien-mga** [comment]
Yeah the error is expected. There's no crash actually, the scene fails loading due to the error, so the game closes (but does not crash - i.e. no segfault).

Error reporting could be improved on scene load errors in general as they tend to be confusing to users, but it's not a bug per se.

**Comment by akien-mga** [comment]
Actually, it's a bug that VisualScript lets you create a node of type EditorSettings which breaks the scene.
